service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/UploadAdditionalApplicationsController.java

```diff
@@ -117,12 +119,19 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmit(@RequestBody Cal
         CaseData caseData = getCaseData(caseDetails);
 
         List<Element<HearingOrder>> unsealedCMOs = caseData.getDraftUploadedCMOs();
-        List<Element<DraftOrder>> draftOrders = caseData.getTemporaryC2Document().getDraftOrdersBundle();
-        unsealedCMOs.addAll(draftOrders.stream()
-            .map(Element::getValue)
-            .map(HearingOrder::from)
-            .map(ElementUtils::element)
-            .collect(Collectors.toList()));
+
+        Optional.ofNullable(caseData.getTemporaryC2Document())
+            .map(C2DocumentBundle::getDraftOrdersBundle)
+            .filter(not(Objects::isNull))
```

Comment:
Can it be simplified as `Objects.nonNull`?

Reply:
done!